workshop/exercise-2/README.md

```diff
@@ -2,26 +2,33 @@
 In this module, you download and install Istio.
 
 1.  Either download Istio directly from [https://github.com/istio/istio/releases](https://github.com/istio/istio/releases) or get the latest version by using curl:
-    ```bash
-    curl -L https://git.io/getLatestIstio | sh -
-    ```
+
+```bash
+curl -L https://git.io/getLatestIstio | sh -
+```
+
 2. Extract the installation files.
 3. Add the `istioctl` client to your PATH. The `<version-number>` is in the directory name. For example, run the following command on a MacOS or Linux system:
-```
+
+```shell
 export PATH=$PWD/istio-<version-number>/bin:$PATH
 ```
+
 4. Change the directory to the Istio file location.
 
 5. Install Istio on the Kubernetes cluster. Istio is deployed in the Kubernetes namespace `istio-system`.
+
 ```bash
-kubectl apply -f install/kubernetes/istio-demo.yaml
+kubectl apply -f <istio-installation>/install/kubernetes/istio-demo.yaml
```

Comment:
"istio-installation" is never defined

Reply:
the export above assumes you have `$PWD` I added `$PWD` to this note to be more verbose.